tests/test.rs

```diff
@@ -95,6 +95,29 @@ fn real_manifest() {
     );
 }
 
+#[test]
+fn real_manifest_parallel() {
+    cargo_hack(["check", "--parallel"])
+        .assert_success("real")
+        .stderr_not_contains(
+            "
+            running `cargo check` on member1
+            running `cargo check` on member2
+            running `cargo check` on member3
+            ",
+        )
+        .stderr_contains("running `cargo check` on real");
+
+    cargo_hack(["check", "--parallel", "--workspace"]).assert_success("real").stderr_contains(
+        "
+        running `cargo check` on member1 (1/4)
+        running `cargo check` on member2 (2/4)
+        running `cargo check` on member3 (3/4)
+        running `cargo check` on real (4/4)
```

Comment:
I don't think the execution order guarantee is very important, so I think it is fine to assert "running <command> on <crate>" and "(N/4)" separately.